ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -283,7 +283,7 @@ import '../nuxeo-button-styles.js';
             on-scroll="_scroll"
           >
             <template>
-              <div class="item">
+              <div class="item" role="cell">
```

Comment:
This div has, as a child, an element called `nuxeo-data-table-row`. So it looks like it is a row, not a cell.